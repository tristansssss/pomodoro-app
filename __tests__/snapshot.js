import React from "react";
import renderer from "react-test-renderer";
import Index from "../pages/index";

it.skip("renders homepage unchanged", () => {
  const tree = renderer.create(<Index />).toJSON();
  expect(tree).toMatchSnapshot();
});
