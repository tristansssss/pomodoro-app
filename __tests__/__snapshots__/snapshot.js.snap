// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders homepage unchanged 1`] = `
.emotion-0 {
  border: 1px solid black;
}

<div
  className="emotion-0"
>
  Home
</div>
`;
